### DISCUSSION TITLE
Bundler: Very WEIRD Side Effect Of Wiki Links Toggle ... ???

### DISCUSSION THREAD

**Discussion by TiddlyTweeter** [wikilabs/plugins#82] [General]
I am not sure you can call this a "bug". It is very odd :-). But can be reproduced as follows ...

- Install an empty
- Install The Bundler plugin
- Go to $:/plugins/wikilabs/bundler/ui/ControlPanel/bundler-config
- UN-CHECK the box for Wiki Links ....
![Screenshot 2021-03-23 192015](https://user-images.githubusercontent.com/28830541/112199133-226e0b00-8c0e-11eb-9ccd-f858da94808d.jpg)
- The StoryView NO LONGER WORKS!!! All three Story Views are DEactivated!
![after - Screenshot 2021-03-23 192205](https://user-images.githubusercontent.com/28830541/112199533-8d1f4680-8c0e-11eb-8629-a9084416d327.jpg)

Maybe this is NOT due to the bundler, rather a broader issue?

But, @pmario I thought you'd be interested as it is reproducible.

Best wishes
TT

**Comment by pmario** [comment]
This is the same for tiddlywiki.com - You'd create an issue there

**Reply by TiddlyTweeter** [reply]
Thanks! I should have checked there first. It just really surprised me that such a bug has gone unnoticed!
I posted an issue at: https://github.com/Jermolene/TiddlyWiki5/issues/5566#issue-839475492

**Reply by TiddlyTweeter** [reply]
This issue is under active resolution here: [TW pull #5573](https://github.com/Jermolene/TiddlyWiki5/pull/5573#issue-602255611)
